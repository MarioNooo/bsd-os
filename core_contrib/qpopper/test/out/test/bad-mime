Date: Thu, 2 Jan 1992 12:21:18 -0800 (PST)
From: Laurence Lundblade <lgl@cac.washington.edu>
Sender: Laurence Lundblade <lgl@cac.washington.edu>
Reply-To: Laurence Lundblade <lgl@cac.washington.edu>
Subject: Pine source distribution
To: Bill Bathurst <bill@cpac.washington.edu>
yyy: extra header
xx: extra header
MIME-Version: 1.0
Content-type: "multipart"
 /
 (slkdfjslkfj) 
 mixed (sdlfkj_ ) ;
 boundary
 =
 "\A\b\C"

extar junklaslkjasdlkj


--AbC
content-type: multipart/alternative; 

--xxx

This is the basic version. Too bad you can't see all the cool stuff

--xxx
content-type: text/enriched

This is is the start of a token: <<


Greetings,
<bold>  This message is to let you know the official Pine 2.0 </bold>source release
date will be Jan 17, 1992. We're sorry we didn't <p>quite make the December
release date. We want to make sure we've got our act together. The
January<p>
<nofill>
This is
a single
line
</nofill>
These

are

on separate

lines

<color><param>
red
</param>What color is this
line<color>

--xxx
content-type: application/xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxyyyyyyyyyyzzzzzzzzzzaaaaaaaaaabbbbbbbbbbcccccccccc
content-disposition: attachment; filename="a123456789b123456789c123456789d123456789e123456789f123456789g123456789h123456789i123456789j123456789.QCP"
content-description: A third alternative....

release is firm and we don't expect it to slip. I'll be sending out
another message on the actual date and you are all on the list. Pine 2.0
is the official source release; you will be able to redistribute freely if
you like.
--xxx--

--AbC
content-type: application/foo-bar
content-description: extra junk

Laurence Lundblade                       206-543-5617
  lgl@cac.washington.edu
     Computing and Communications, U of Washington, Seattle

--AbC
content-type: multipart/alternative; boundary=xxu

--xxu
content-type: TEXTtext/enriched

This is is the start of a token: <<


Greetings,
<bold>  This message is to let you know the official Pine 2.0 </bold>source release
date will be Jan 17, 1992. We're sorry we didn't <p>quite make the December
release date. We want to make sure we've got our act together. The
January<p>
<nofill>
This is
a single
line
</nofill>
These

are

on separate

lines

<color><param>
red
</param>What color is this
line<color>

--xxu
content-type: mdel/foogoo

lksdjflksdjflksjdflkjsf
sdfkjlsdlkfjdslkjfdslkj
sdlkfjdslkjf

--xxu--

--AbC--





