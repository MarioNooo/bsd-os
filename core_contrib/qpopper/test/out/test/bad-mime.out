Date: Thu, 2 Jan 1992 12:21:18 -0800 (PST)
From: Laurence Lundblade <lgl@cac.washington.edu>
Subject: Pine source distribution
To: Bill Bathurst <bill@cpac.washington.edu>


[ invalid MIME formatting; part omitted ]


[ foo-bar attachment omitted ]


[ unknown attachment omitted ]

